.zsh-autosuggestionsrc

```diff
@@ -0,0 +1,13 @@
+# Setup zsh-autosuggestions
+source $HOME/.zsh-autosuggestions/autosuggestions.zsh
+
+# Enable autosuggestions automatically
+zle-line-init() {
+    zle autosuggest-start
+}
+
+zle -N zle-line-init
+
+# use ctrl+t to toggle autosuggestions(hopefully this wont be needed as
+# zsh-autosuggestions is designed to be unobtrusive)
+bindkey '^T' autosuggest-toggle
```

Comment:
@ronjouch sure, I just took what the existing install script does without changes. Shall I make an update?